community-operators/dynatrace/dynatrace-monitoring.v0.2.0.clusterseviceversion.yaml

```diff
@@ -0,0 +1,215 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      {
+      	"apiVersion": "dynatrace.com/v1alpha1",
+      	"kind": "OneAgent",
+      	"metadata": {
+      		"name": "oneagent",
+      		"namespace": "dynatrace"
+      	},
+      	"spec": {
+      		"apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api",
+      		"skipCertCheck": false,
+      		"tokens": "",
+      		"nodeSelector": {},
+      		"tolerations": [
+      			{
+      				"effect": "NoSchedule",
+      				"key": "node-role.kubernetes.io/master",
+      				"operator": "Exists"
+      			}
+      		],
+      		"image": "",
+      		"args": [
+      			"APP_LOG_CONTENT_ACCESS=1"
+      		],
+      		"env": []
+      	}
+      }
+    categories: "Monitoring"
```

Comment:
We may want to add the `OpenShift Optional` category. 

Does anyone see other categories that should be added as well? A list of available categories can be found [here](https://github.com/operator-framework/community-operators/blob/b32a24e7a9745c20010d565223205f5b0b3bd20a/docs/required-fields.md#categories)